recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -0,0 +1,216 @@
+        <slot name="reactions" />
+      </div>
+      <!-- @slot Slot for threading row component -->
+      <slot name="threading" />
+    </template>
+
+    <!-- Action menu -->
+    <template #right>
+      <div
+        v-show="isActive"
+        class="d-ps-absolute d-tn16 d-r12"
+      >
+        <dt-lazy-show
+          :appear="true"
+          transition="fade"
+          :show="isActive"
+        >
+          <!-- @slot Slot for actions hover menu component -->
+          <slot name="menu" />
+        </dt-lazy-show>
+      </div>
+    </template>
+  </dt-list-item>
+</template>
+
+<script>
+import {} from './feed_item_row_constants.js';
+import { DtAvatar } from '@/components/avatar';
+import { DtLazyShow } from '@/components/lazy_show';
+import { DtListItem } from '@/components/list_item';
+
+export default {
+  name: 'DtRecipeFeedItemRow',
+
+  components: {
+    DtAvatar,
+    DtLazyShow,
+    DtListItem,
+  },
+
+  inheritAttrs: false,
+
+  props: {
+    /**
+     * Show avatar, show header text or dont show left time and vice versa
+     */
+    showHeader: {
+      type: Boolean,
+      default: false,
+    },
+
+    /**
+     * Show the avatar of the user, if this is not passed in, the initials would display
+     */
+    avatarImageUrl: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * The display name of the sender
+     */
+    displayName: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * time string displayed as is.
+     * Shown on the header when showHeader is true
+     */
+    time: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * short time string without AM/PM displayed as is.
+     * Shown on the left of feed item when showHeader is false and isActive is true
+     */
+    shortTime: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * displays a darkened background on the row.
+     */
+    isActive: {
+      type: Boolean,
+      default: false,
+    },
+
+    // TODO: implement as part of DT-1157
+    // state: {
+    //   type: String,
+    //   default: 'normal',
+    // },
+  },
+
+  emits: [
+    /**
+     * Fires when hovered over feed row
+     *
+     * @event hover
+     * @type {Boolean}
+     */
+    'hover',
+
+    /**
+     * Fires when focused over feed row
+     *
+     * @event focus
+     * @type {Boolean}
+     */
+    'focus',
+  ],
+
+  computed: {
+    avatarInitials () {
+      const name = this.displayName.split(' ');
+      if (name.length >= 2) {
+        return name[0].charAt(0) + name[1].charAt(0);
+      } else if (name && name[0].charAt(0) != null) {
+        return name[0].charAt(0);
+      }
+
+      return '';
+    },
+
+    feedListeners () {
+      return {
+        ...this.$listeners,
+        mouseenter: _ => this.setHover(true),
+        mouseleave: _ => this.setHover(false),
+        focusin: _ => this.setFocus(true),
+        focusout: _ => this.setFocus(false),
+      };
+    },
```

Comment:
Im guessing you could use arrow functions to avoid define anonymous functions as event listeners, sth like this
(I didn't tested it)

feedListeners () {
      return {
        ...this.$listeners,
        mouseenter: () => this.setHover(true),
        mouseleave: () => this.setHover(false),
        focusin: () => this.setFocus(true),
        focusout: () => this.setFocus(false),
      };
    },